### ISSUE TITLE
Wallet crash during syncing

### ISSUE THREAD

**Issue by FriedCircuits** [PaycoinFoundation/paycoin#73]
I think I found a bug.

I was re-syncing my wallet since I updated late and if you hit the minting tab while syncing it will crash the wallet. Tested it twice. After starting the wallet it again it will continue syncing fine. Just had to do a repairwallet after syncing to get back into business.

**Comment by IngCr3at1on** [comment]
The repairwallet part was expected but otherwise I cannot replicate this bug (wondering if it's specifically because of the late update; showing invalid minting transactions from running while the wallet was out of date). I tested this on my linux box, so the other possibility is that this could be an OS specific bug.

@FriedCircuits was that on Windows?
@PaycoinMaster have you heard of any similar issues?

**Comment by FriedCircuits** [comment]
Hi,
Yes, it was on Win7x64. I haven't tried recently to see if the problem still happens.

**Comment by IngCr3at1on** [comment]
@FriedCircuits I believe I saw something from @mitchellcash saying he had a similar issue in OS X (still nothing I've seen in Linux).

I really wish windows through return output to cmd if you opened the app from there and it crashed but I don't believe it will. Any pertinent error logs will be somewhere in the Event Viewer for windows (if you do duplicate it I mean).

**Comment by mitchellcash** [comment]
Thats right, I commented in #176 as I experienced the exact same issue. In hindsight this may have been the better issue thread to comment on.

**Comment by IngCr3at1on** [comment]
Had a scroll through the log but off the top of my head I'm not sure; I'll take a closer look later though.

**Comment by IngCr3at1on** [comment]
Does anyone know if this is still occurring?
